packages/theme-data/src/basics/colors.js

```diff
@@ -152,5 +152,7 @@ export default {
   // Alert colors
   error: { value: "#ec4a41", type: COLOR },
   success: { value: "#87b340", type: COLOR },
-  warning: { value: "#faa21b", type: COLOR }
+  warning: { value: "#faa21b", type: COLOR },
+
+  textLinkLightGrayTextColor: { value: "#006EAF", type: COLOR }
```

Comment:
@wmui51 i've added this, not sure if the naming scheme is the best

Reply:
I don't think we need to put color specific but just light and dark themes would probably work better, especially since dark grey/blue uses the same color. So something like `textlink.light.color` and `textlink.dark.color`. Also you can add #6dd2ff from the dark themes to this as well.